### ISSUE TITLE
Please, STOP autocreating devices!!

### ISSUE THREAD

**Issue by indigo-jay** [kw123/Hue-Lights-Indigo-plugin#24]
Please, **PLEASE**, stop autocreating devices every time the plugin starts up. There are very good reasons why I don't want those devices sticking around.

Autocreation of devices is not standard practice for any of the built-in plugins, and should not be done by any other plugins in order to present a consistent user experience for the device lifecycle. If you want to offer a menu item that will do it, that's great. But doing it every time the plugin starts is just inconsistent with everything else in Indigo.

**Comment by kw123** [comment]
There should be an option in config to switch that off KarlOn 20.09.2023, at 22:59, Jay Martin ***@***.***> wrote:﻿
Please, PLEASE, stop autocreating devices every time the plugin starts up. There are very good reasons why I don't want those devices sticking around.
Autocreation of devices is not standard practice for any of the built-in plugins, and should not be done by any other plugins in order to present a consistent user experience for the device lifecycle. If you want to offer a menu item that will do it, that's great. But doing it every time the plugin starts is just inconsistent with everything else in Indigo.

**Comment by indigo-jay** [comment]
Yeah, I did find that. I don't think the default should be to automatically create them..

**Comment by kw123** [comment]
I can change the default. But several folks really liked it.I shall follow your advice out of town - back on Tuesday. Karl On 22.09.2023, at 17:39, Jay Martin ***@***.***> wrote:﻿
Yeah, I did find that. I don't think the default should be to automatically create them..

**Comment by kw123** [comment]
default has been set to OFF and the plugin is updated on GitHub and indigo
